tests/features/bootstrap/FeatureContext.php

```diff
@@ -85,6 +86,14 @@ public function aUserIdentifiedByWithAVettedToken($institution, $nameId, $tokenT
         }
     }
 
+    /**
+     * @Given /^a user from "([^"]*)" identified by "([^"]*)"$/
+     */
+    public function aUserIdentifiedBy($institution, $nameId)
+    {
+        // For now, do nothing, this user is not known to Gateway, but might exist.
```

Comment:
If not used please remove. This only will make the Behat tests less readable if something states X will happen, but this isn't actually done.

Reply:
Yes, agreed, removing the step definition and its usages